dj_gui_api_server/DJGUIAPIServer.py

```diff
@@ -219,6 +221,31 @@ def insert_tuple(jwt_payload: dict):
         return str(e), 500
 
 
+@app.route('/api/record/dependency', methods=['GET'])
+@protected_route
+def record_dependency(jwt_payload: dict) -> dict:
+    """
+    Route to insert record. Expects:
+        (html:GET:Authorization): Must include in format of: bearer <JWT-Token>
+        (html:query_params): {"schema_name": <schema_name>, "table_name": <table_name>,
```

Comment:
```suggestion
        (html:query_params): {"schemaName": <schema_name>, "tableName": <table_name>,
```
maybe change to match the format that's used on the frontside?